Tests/GitHubRepositories.tests.ps1

```diff
@@ -829,970 +741,1060 @@ try
     }
 
-    Describe 'GitHubRepositories\Get-GitHubRepositoryTeamPermission' {
-        BeforeAll {
-            $repositoryTeamPermissionTypeName = 'GitHub.RepositoryTeamPermission'
-            $repoName = [Guid]::NewGuid().Guid
-            $repo = New-GitHubRepository -OrganizationName $script:organizationName -RepositoryName $repoName
+    Context "When specifiying the 'URI' Parameter from the Pipeline" {
+        It 'Should not throw' {
+            { $repo | Set-GitHubRepositoryActionsPermission -AllowedActions 'All' } |
+            Should -Not -Throw
+        }
+    }
 
-            $teamName = [Guid]::NewGuid().Guid
-            $description = 'Team Description'
-            $privacy = 'closed'
-            $MaintainerName = $script:ownerName
+    AfterAll {
+        if (Get-Variable -Name repo -ErrorAction SilentlyContinue)
+        {
+            $repo | Remove-GitHubRepository -Force
+        }
+    }
+}
 
-            $newGithubTeamParms = @{
-                OrganizationName = $script:organizationName
-                TeamName = $teamName
-                Description = $description
-                Privacy = $privacy
-                MaintainerName = $MaintainerName
-            }
+Describe 'GitHubRepositories\Get-GitHubRepositoryTeamPermission' {
+    BeforeAll {
+        $repositoryTeamPermissionTypeName = 'GitHub.RepositoryTeamPermission'
+        $repoName = [Guid]::NewGuid().Guid
+        $repo = New-GitHubRepository -OrganizationName $script:organizationName -RepositoryName $repoName
 
-            $team = New-GitHubTeam @newGithubTeamParms
+        $teamName = [Guid]::NewGuid().Guid
+        $description = 'Team Description'
+        $privacy = 'closed'
+        $MaintainerName = $script:ownerName
 
-            $permissions = 'Push', 'Pull', 'Maintain', 'Triage', 'Admin'
+        $newGithubTeamParms = @{
+            OrganizationName = $script:organizationName
+            TeamName = $teamName
+            Description = $description
+            Privacy = $privacy
+            MaintainerName = $MaintainerName
         }
 
-        Foreach ($permission in $permissions) {
-            Context "When the Team Permission is $permission" {
-                BeforeAll {
-                    $setGitHubRepositoryTeamPermissionParms = @{
-                        Uri = $repo.svn_url
-                        TeamSlug = $team.slug
-                        Permission = $permission
-                    }
-
-                    Set-GitHubRepositoryTeamPermission @setGitHubRepositoryTeamPermissionParms
+        $team = New-GitHubTeam @newGithubTeamParms
 
-                    $getGithubRepositoryTeamPermissionParms = @{
-                        Uri = $repo.svn_url
-                        TeamSlug = $team.slug
-                    }
-
-                    $repoPermission = Get-GitHubRepositoryTeamPermission @getGithubRepositoryTeamPermissionParms
-                }
+        $permissions = 'Push', 'Pull', 'Maintain', 'Triage', 'Admin'
```

Comment:
```suggestion
```

Looks like we either drop that, or reference it at 1510 (I think dropping it is fine).